Nos/Views/WarningView.swift

```diff
@@ -115,10 +115,11 @@ struct OverlayContentReportView: View {
                         .scaledToFit()
                         .frame(width: 48, height: 48) // Set the width and height to 48
                         .padding(.bottom, 20)
-                    if controller.authorReports.count > 0 {
-                        ContentWarningMessage(reports: controller.authorReports, type: "author")
-                    } else if controller.noteReports.count > 0 {
-                        ContentWarningMessage(reports: controller.noteReports, type: "note")
+                    
+                    if controller.noteReports.count > 0 {
+                        ContentWarningMessage(reports: controller.noteReports, type: .note)
+                    } else if controller.authorReports.count > 0 {
+                        ContentWarningMessage(reports: controller.authorReports, type: .author)
```

Comment:
I reversed the logic here. We never had a discussion or requirements around this and this made more sense to me. If there are reports for a note and its author we now give a message based on the reports for the note since they are more specific.